docs/getting-started.md

```diff
@@ -0,0 +1,96 @@
+Getting Started
+===============
+
+Repository Requirements
+-----------------------
+
+VFS for Git will work with any Git service that supports the
+[GVFS protocol](/Protocol.md). For example, you can create a repo in
+[Azure DevOps](https://azure.microsoft.com/services/devops/), and push
+some contents to it. There are two constraints:
+
+  * Your repo must not enable any clean/smudge filters
+  * Your repo must have a `.gitattributes` file in the root that includes

```

Comment:
Is it "that includes" or "whose contents consists of"?

Reply:
I _think_ there are some `.gitattributes` options that can be added in addition to this one. I don't think an exact value is important.